pkg/network/usm/shared_libraries_test.go

```diff
@@ -620,7 +620,7 @@ func initEBPFProgram(t *testing.T) *ddebpf.PerfHandler {
 				EditorFlag: manager.EditMaxEntries,
```

Comment:
I would expect a change in http/protocol.go but seems ebpf-manager doesn't care about the hash type in the filter
https://github.com/DataDog/datadog-agent/blob/main/pkg/network/protocols/http/protocol.go#L88

Reply:
Yes, I was surprised to realize that this was left with the wrong type when we originally migrated from HashMap to LRU map. Now that we're going back to things as they were we don't need to change anything 😅